baystation12.dme

```diff
@@ -3187,6 +3187,13 @@
 #include "maps\random_ruins\space_ruins\space_ruins.dm"
 #include "maps\torch\torch.dm"
 #include "maps\torch\torch_define.dm"
+#include "maps\torch\torch1_deck5.dmm"
+#include "maps\torch\torch2_deck4.dmm"
+#include "maps\torch\torch3_deck3.dmm"
+#include "maps\torch\torch4_deck2.dmm"
+#include "maps\torch\torch5_deck1.dmm"
+#include "maps\torch\torch6_bridge.dmm"
+#include "maps\torch\torch_z1_admin.dmm"
```

Comment:
This was included at runtime and should have been ignored by the .gitignore file, how did this get through? Startup times will be 2x because it redefines the entire map once at initialization then again at runtime.

Reply:
It's included here instead: https://github.com/Skyrat-SS13/Vesta.Bay/blob/1045c1281f1aad84e6e83c8e9616b3e3a0cd6f79/maps/torch/torch.dm#L145-L152